wave_containers/include/wave/containers/measurement.hpp

```diff
@@ -0,0 +1,38 @@
+#ifndef WAVE_CONTAINERS_MEASUREMENT_HPP
+#define WAVE_CONTAINERS_MEASUREMENT_HPP
+
+#include <chrono>
+
+namespace wave {
+
+using TimeType = std::chrono::steady_clock::time_point;
+
+/** Generic measurement storage type designed to be used with
+ * MeasurementContainer. */
+template<typename T, typename S>
+struct Measurement {
+    TimeType time_point;
+    S sensor_id;
+    T value;
```

Comment:
Yes. You want the sensor_id type to be an enum class such as in the test:

```cpp
enum class SomeSensors { S1, S2, S3 };
```

That way you can't ask for Lidar #2 when you only have one Lidar.